pageserver/src/layered_repository.rs

```diff
@@ -712,13 +712,15 @@ impl LayeredRepository {
         let mut conf_content = r#"# This file contains a specific per-tenant's config.
 #  It is read in case of pageserver restart.
 
-# [tenant_config]
+[tenant_config]
 "#
         .to_string();
 
         // Convert the config to a toml file.
         conf_content += &toml_edit::easy::to_string(&tenant_conf)?;
 
+        info!("conf_content {}", conf_content);
```

Comment:
Not sure if this a good idea. If we add any sensitive data ie. tokens/keys to tenant conf they'll get logged too. May be make it a debug.

Reply:
We have an example of that in proxy code, if needed we can make a custom display that omits sensitive fields